packages/react/src/Card/Card.tsx

```diff
@@ -62,31 +63,31 @@ const CardRoot = forwardRef<HTMLAnchorElement, CardProps>(
         setIsHovered(!isHovered)
         onMouseEnter?.(event)
       },
-      [onMouseEnter, isHovered],
```

Comment:
These code style changes are not intentional - this is just how Prettier auto-formatted when I saved the file.